pennylane/templates/subroutines/fable.py

```diff
@@ -0,0 +1,177 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the template for the Fast Approximate BLock Encoding (FABLE) technique. 
+It can synthesize quantum circuits for approximate block-encodings of matrices, which
+the embedding of a matrix in the leading block of a larger unitary matrix.
+"""
+import warnings
+import numpy as np
+import pennylane as qml
+from pennylane.templates.state_preparations.mottonen import compute_theta, gray_code
+from pennylane.operation import AnyWires, Operation
+from pennylane.wires import Wires
+
+
+class FABLE(Operation):
```

Comment:
Please could we make sure that `FABLE` is accessible in the documentation

Reply:
Yes, sorry, I've fixed the docstrings and added it to the documentation.